.gitignore

```diff
@@ -0,0 +1,8 @@
+
+# Ignore these generated files so they dont get checked in again
+config.ini
+generate-sol-log.sh
+post-deploy.sh
+test.sh
+pre-deploy.sh
+vnc-record.sh
```

Comment:
Actually, it is an underscore rather than a dash.  I got it wrong in the comment.  
vnc_record.sh